bolt_transport.go

```diff
@@ -255,6 +255,11 @@ func (t *BoltTransport) dispatchHistory(s *Subscriber, toSeq uint64) {
 			}
 		}
 		s.HistoryDispatched(responseLastEventID)
+		if !afterFromID {
+			if c := t.logger.Check(zap.ErrorLevel, "Can't find LastEventID in Bolt DB"); c != nil {
```

Comment:
```suggestion
			if c := t.logger.Check(zap.ErrorLevel, "Can't find requested LastEventID"); c != nil {
```

No need to mention Bolt DB, so we can log the exact same messages for other transports and ease switch between transports.

Reply:
Also, the `Info` level will be good enough, WDYT? (Error would mean waking up ops just for a client providing a bad ID).